2021/2021-09-17.md

```diff
@@ -0,0 +1,100 @@
+# Infrastructure Meeting 2021-09-17
+
+## Present
+
+- Dan Trickey
+- Jake Howard
+- James Seden Smith
+- Karina Kwiatek
+- Peter Law
+- Tyler Ward
+
+## Topics
+
+### External comms
+
+- Do we have anything to add to<https://github.com/srobo/volunteer-emails/pull/44>?
+  - Action: James: add something about us having this meeting
+
+### Terms of reference
+
+- Not fully defined yet, but hopefully will be soon
+- Still awaiting admin access to GitHub G-suite etc
+
+### How often should we meet?
+
+- We probably want a mix of Meetings & Doings
+  - try to keep Meetings for general direction and coordination, Doings for more technical things & getting stuff done
+- Monthly
+- If we do a lot of things as infrastructure-as-code, we can do several things async
+- Maybe sooner than a month if we get our terms of reference sorted sooner
+- Decide the next meeting at the end of the previous?
+  - Good in practice, has some issues
+  - Let’s just put things in the calendar, and adjust if we have an issue
+
+### Volunteer _CRM_?
+
+- Would a CRM or similar aid our management of G-Suite and/or GitHub etc.?
+- Not sure where this information is right now?
+- Could allow us to track when volunteers volunteer (perhaps more useful for less frequent volunteers) that sort of thing
+- Perhaps something for an all committees meeting?
+
+### When should we meet?
+
+- Fridays seem to be good for everyone in the meeting; by virtue of the other days of the week being non-doable for some
+- Maybe go a bit earlier in the evening? 18:30 perhaps?
+- Ideally should poll people not in the meeting to avoid survivorship bias
+- Action: Peter: put a Doodle in chat to get input from everyone who might want to come, then set a recurring date in the calendar.
+- Try to be async from the all-committee meetings
+
+### What should we do? (Let’s build a backlog)
+
+- A lot of what falls in our remit is things which has historically sat between teams
+
+- What services are we responsible for?
+
+  - G-Suite
+  - Mailchimp
+  - Github
+  - DNS
+  - (Website) Nginx reverse proxy
+  - Digital Ocean (kubernetes, project)
+  - website/docs/etc. in so far as the common platform -- e.g: Jekyll maintenance, GitHub Pages
+  - others? We should find out early on.
+
+- Where does userman go?
+
+  - It’s purely for managing competitor users by team leaders, so we’re expecting that it’s a competition team thing
+  - Action: Jake: check that the competition team are aware of this
+
+- We’re not intending to run a managed service, but we’re also not a VPS provider
+
+  - Security updates etc
+  - Ensure the machines
```

Comment:
Think this may have been cut off?

Reply:
:( https://github.com/srobo/infrastructure-team-minutes/pull/1/commits/10984fdb9294e0e05c0901845cf25ec9ec43823f expands this section and adds more context.